tiny_dnn/activations/activation_function.h

```diff
@@ -165,7 +196,16 @@ class tan_h : public function {
   x));
   }*/
 
-  float_t df(float_t y) const override { return float_t(1) - sqr(y); }
+  inline float_t df(float_t y) const override { return float_t(1) - sqr(y); }
+  void itedf(vec_t &cur,
+             const vec_t &prev,
+             const vec_t &out,
+             size_t cnt) const {
+    for (size_t i = 0; i < cnt; i++) {
+      cur[i] = prev[i] * df(out[i]);
```

Comment:
Since there is a curly bracket, it doesn't align cur[i], since it's not consequentive assignmentsanymore. As I said, putting curly brackets seems like right thing to do and has nothing to do with clang-tool. Yet aligning of assignments after `for` is probably a bug